### PR TITLE
grid catalog v1

### PR DESCRIPTION
**Pull request by edgarlokys**
https://edgarlokys.github.io/layout_catalog_grid/

### CODE REVIEW

File: src/styles/utils/_media.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (max-width: $min-size-media) {
+  .catalog {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
```
Comment (IvanFesenko) [inline src/styles/utils/_media.scss:17]: It's bad practice use separate file with media while using sass. Move it inside catalog selector

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Check my comment

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@import "./../utils/mixin";
+@import "./../utils/variables";
+
+.catalog {
```
Comment (tetiana-ivanchenko) [inline src/styles/blocks/catalog.scss:4]: there is example from official documentstion
media query should be inside selector's block
image

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss (hunk 2 above)
Comment (edgarlokys) [reply src/styles/blocks/catalog.scss:4]: i think fixed it

Comment (edgarlokys) [reply src/styles/blocks/catalog.scss:4]: check plz 'correct v2'

File: src/styles/utils/_media.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@media (min-width: $medium-size-media) and (max-width: $max-size-media) {
```
Comment (mykhalenych) [inline src/styles/utils/_media.scss:1]: why do you duplicate this code?
delete here and use only in catalog file

File: src/styles/utils/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+$max-size-media: 1024px;
+$medium-size-media: 768px;
+$min-size-media: 488px;
+
+$card-width: 200px;
+
+$border-color: #ccc;
+$header-item-color: #060b35;
```
Comment (mykhalenych) [inline src/styles/utils/_variables.scss]: Screenshot 2023-03-07 at 17 29 44
check your configs rules, you missed "add new line at the end"

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/styles/utils/_media.scss (hunk 3 above)
Comment (VitaliyBondarenko1982) [reply src/styles/utils/_media.scss:1]: Please, fix comments about remove duplicate code. Remove this file, you already use it in catalog

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  @media (min-width: $min-size-media) and (max-width: $medium-size-media) {
+    .catalog {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+  
+  @media (max-width: $min-size-media) {
+    .catalog {
+      grid-template-columns: repeat(1, $card-width);
+    }
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/catalog.scss:26]: You can simplify it. Check if it'll work
```suggestion
   @media (min-width: $min-size-media) {
    .catalog {
      grid-template-columns: repeat(2, $card-width);
    }
  }
  
  @media (min-width: $medium-size-media) {
    .catalog {
      grid-template-columns: repeat(3, $card-width);
    }
  }
  
  @media (min-width:  $max-size-media) {
    .catalog {
      grid-template-columns: repeat(4, $card-width);
    }
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- The previous comments are not fixed. Maybe you forgot to commit and push changes? If you have any questions, please, ask them in the fe_chat 🙏 

- Align the cards container to the center 
image

- There should be four columns starting at 1024px
image

- There should be three columns starting at 768px
image

- Same with other queries. Check it, please

**Pull request closed without merging**